react/features/base/participants/functions.js

```diff
@@ -271,6 +289,20 @@ export function getParticipantDisplayName(
     return defaultRemoteDisplayName;
 }
 
+/**
+ * Returns screenshare participant's display name.
+ *
+ * @param {(Function|Object)} stateful - The (whole) redux state, or redux's {@code getState} function to be used to
+ * retrieve the state.
+ * @param {string} id - The ID of the screenshare participant's display name to retrieve.
+ * @returns {string}
+ */
+export function getScreenshareParticipantDisplayName(stateful: Object | Function, id: string) {
+    const owner = getParticipantById(stateful, getFakeScreenShareParticipantOwnerId(id));
+
+    return `${owner.name}'s screen`;
```

Comment:
It doesn't seem like we have the ability to persist the name for our virtualScreenshareParticipants. Can we assume the display name will always be owner.name's screen?

Reply:
Yes, it is safe to assume that it will always be owner.name's screen.

Reply:
Probably we need to add this to i18n at some point, there are people that will complain about the English word in there :)

Reply:
Good point, thanks for bringing it up @damencho. @WillLiang918 can you pls add it to your list, thanks!